frontend/lib/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -310,6 +310,7 @@ class Multiselect extends React.PureComponent<Props, State> {
               Placeholder: {
                 style: () => ({
                   flex: "inherit",
+                  opacity: "0.7",
```

Comment:
**question (non-blocking):** Is it worthwhile to put this value in a shared place? I see that we utilize `0.7` [here](https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/widgets/TextInput/TextInput.tsx#L264-L266) and [here](https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/widgets/TextArea/TextArea.tsx#L239-L241). Sometimes putting these constants in a shared place help not just for ease of updating, but for code documentation purposes.

Reply:
Fair question, not seeing a specific theme primitive file that jumps out to add this into, but think we probably should have either the `opacity` or `placeholderColor` in theme for consistency across elements discussed in this PR (TextInput, TextArea, MultiSelect) and others like ChatInput, NumberInput, etc.